### ISSUE TITLE
Prefix text not showing for clojure commands

### ISSUE THREAD

**Issue by filipesilva** [hlissner/doom-emacs#5762]
### What did you expect to happen?

When using clojure, I expected prefixes for the commands after pressing localleader.

They seem to be defined here https://github.com/hlissner/doom-emacs/blob/b4cc1c84d8e77e3a5cb2175be4908b6ce46d93a6/modules/lang/clojure/config.el#L133-L143

### What actually happened?

I only see `+prefix` instead of the actual prefixes.

![image](https://user-images.githubusercontent.com/4172079/141289674-f006e6e9-4e16-4ca0-9837-a40d913f113e.png)

### Steps to reproduce

1. use the `clojure` doom emacs lang module
2. open a clj file
3. press localleader and wait for the shortcuts to appear

### System Information

https://pastebin.com/mKUEZwZ3

**Comment by hlissner** [comment]
I can't reproduce this in vanilla Doom. 

![image](https://user-images.githubusercontent.com/510883/143308330-9d9e9071-2091-4691-9ad7-a660dbff017f.png)

Please double check your private config and try to reproduce it [without your private config](https://github.com/hlissner/doom-emacs/blob/develop/docs/getting_started.org#testing-in-dooms-sandbox).

**Comment by filipesilva** [comment]
Thanks for taking the time!

I was on my way to follow those instructions and ended up finding out the trigger: `.clj` and `.cljs` show the text, but `.cljc` do not.

So the real repro is to open a `cljc` file, not a `clj` one. Sorry about that!

**Comment by hlissner** [comment]
This issue should now be resolved. Let me know if that isn't the case and I'll reopen it. Thanks for bringing it to my attention!

**Comment by filipesilva** [comment]
I upgraded via `doom upgrade`, ran `doom sync`, restarted emacs, and I can see in `~/.emacs.d/modules/lang/clojure/config.el` that your changes are there.

I found a bit odd that you added just `clojurec-mode` instead of `*-mode-map` like the others. I looked around and did find `clojurec-mode-map` in https://github.com/clojure-emacs/clojure-mode/blob/7d3c0c16e4aa14a051b393c249f0f4d307a2c74d/clojure-mode.el#L3063. 

Edited the file locally, sync, reload, and it seems to work. Opened a PR, let me know if it looks ok!
